values.yaml

```
@@ -643,11 +643,12 @@ ingester:
       path: /ready
       port: http-metrics
       scheme: HTTP
-  livenessProbe:
-    httpGet:
-      path: /ready
-      port: http-metrics
-      scheme: HTTP
+  ## Liveness probes for ingesters are not recommended.
+  ## Ref: https://cortexmetrics.io/docs/guides/running-cortex-on-kubernetes/#take-extra-care-with-ingesters
+  # livenessProbe:
+  #   httpGet:
+  #     path: /ready
+  #     port: http-metrics
```

Personally I don't like comments like these within the values.yml. I just think its confusing and clutters the file alot.

May I suggest something like this? (adopted for helm-docs. See: #227 )
```suggestion
  # -- Liveness probes for ingesters are not recommended.
  # Ref: https://cortexmetrics.io/docs/guides/running-cortex-on-kubernetes/#take-extra-care-with-ingesters
  livenessProbe:
```

---

might also work with
```yaml
livenessProbe: []
```

---

It's an object so it should be `{}`.
```suggestion
  # -- Liveness probes for ingesters are not recommended.
  #  Ref: https://cortexmetrics.io/docs/guides/running-cortex-on-kubernetes/#take-extra-care-with-ingesters
  livenessProbe: {}
  #   httpGet:
  #     path: /ready
  #     port: http-metrics
```

---

right.. mea culpa